src/main/java/com/worldheatmap/WorldHeatmapPlugin.java

```diff
@@ -114,15 +112,16 @@ WorldHeatmapConfig provideConfig(ConfigManager configManager) {
 
     @SneakyThrows
     protected void loadHeatmaps() {
-        log.debug("Loading heatmaps under user ID " + mostRecentLocalUserID + "...");
+        log.debug("Loading most recent heatmaps under user ID {}...", mostRecentLocalUserID);
 
+
+        File filepathUserID = HeatmapFile.getLastHeatmap(mostRecentLocalUserID);
```

Comment:
A check should be made inside HeatmapFile.getLastHeatmap() to see if the file `Heatmap Files\userId.heatmaps` exists, because this is where .heatmaps files are currently saved to/loaded from, so existing users will have their progress saved here. If it does exist, then move the file to its new proper location in `Heatmap Files\userID\yyyy-mm-dd.heatmaps` before returning it as a File

Reply:
Correct, I'll do it tonight